src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx

```diff
@@ -29,6 +29,16 @@ To add support for [ProGuard](http://proguard.sourceforge.net/manual/usage.html)
    -keep class com.newrelic.** { *; }
    -dontwarn com.newrelic.**
    -keepattributes Exceptions, Signature, InnerClasses, LineNumberTable, SourceFile, EnclosingMethod
+
+  ##
+  ## NewRelic Gradle plugin 7.x requires the following additions:
+  ##
+  # Retain generic signatures of TypeToken and its subclasses with R8 version 3.0 and higher.
```

Comment:
Please create link of "R8 Version 3.0" with "https://r8.googlesource.com/r8/+/refs/heads/master/compatibility-faq.md#r8-full-mode"

Reply:
Scratch that: line should read "if R8 version 3.0 full-mode is enabled", using the link provided above.